test_efs/dsgrid_project/datasets/modeled/comstock/dataset_with_dimension_ids.json5

```diff
@@ -32,42 +32,42 @@
   },
   dimension_references: [
     {
-      dimension_id: "us_counties_2010_-_comstock_only__b913be60-d3da-407b-935b-6a1b8ee7d92c",
```

Comment:
This is an important change to call out. In the old way of doing things we could handle regenerating a registry by parsing these strings to separate dimension names and UUIDs, and then update the value with what is in the new registry. That isn't possible with the registry database, which stores dimension IDs as integers, and so I changed this format to use `name` explicitly. In order to use this file you have to run a script that converts it.